deployment/data/nginx/app.conf.template

```diff
@@ -0,0 +1,62 @@
+upstream core_backend {
+    server core_backend:8000 fail_timeout=0;
+}
+
+upstream admin_app {
+    server admin_app:3000 fail_timeout=0;
+}
+
+server {
+    listen 80;
+    server_name ${DOMAIN};
```

Comment:
This is the file we should update if we ever want to change how nginx works. The `app.conf` is generated after replacing ${DOMAIN} with the value from `.env.nginx`. This happens in the docker-compose. I'll highlight where.

Reply:
Should we keep a note of this in the docs? Sounds like the kind of thing that could get confusing for a newcomer.

Reply:
It's detailed dev stuff for if you are making changes to the reverse proxy.
We don't need to put it in the public docs but good for us to know.